course/src/isometry.cc

```diff
@@ -307,6 +312,110 @@ const Matrix3 Matrix3::kZero =
 const Matrix3 Matrix3::kOnes = 
     Matrix3(Vector3(1, 1, 1), Vector3(1, 1, 1), Vector3(1, 1, 1));
 
+// Isometry class methods
+
+Isometry Isometry::FromTranslation(const Vector3& v) {
+    return Isometry(v);
+}
+
+Isometry Isometry::RotateAround(const Vector3& v, const double rotation_angle) {
+
+    const double cangle{std::cos(rotation_angle)};
+    const double sangle{std::sin(rotation_angle)};
+
+    Matrix3 rot_matrix;
+
+    if (v == Vector3::kUnitX) {
```

Comment:
This should work for any vector. See [this](https://en.wikipedia.org/wiki/Rotation_matrix#Rotation_matrix_from_axis_and_angle). You'll have to make sure the vector is a unit vector.

Reply:
Addressed!